crypto/tbs/src/lib.rs

```diff
@@ -194,50 +194,44 @@ pub fn verify(msg: Message, sig: Signature, pk: AggregatePublicKey) -> bool {
     pairing(&msg.0, &pk.0) == pairing(&sig.0, &G2Affine::generator())
 }
 
-#[cfg(test)]
-mod tests {
-    use std::collections::BTreeMap;
+pub fn dealer_keygen(
```

Comment:
Did you want to touch that crypto code here or is this a rebase conflict that got resolved the wrong way? @joschisan refactored `tbs` recently: #3468.

Reply:
Joscha made a good observation: you don't need a valid signature for this test, so you can pick a random point and don't need `dealer_keygen`. E.g.:

```rust
Signature(G1Affine::generator())
```

Reply:
Just a rebase conflict, I'll fix with joshican's suggestion

Reply:
Fixed, no more `crypto` changes